db/models.py

```diff
@@ -17,11 +22,16 @@
 
 
 class Movie(models.Model):
-    title = models.CharField(max_length=255)
+    title = models.CharField(max_length=255, )
```

Comment:
There is a trailing comma in the 'title' field definition of the 'Movie' model. This is unnecessary and should be removed for clarity.